### PR TITLE
Tidy up codebase after experimenting with glue's WCSCoordinates

### PR DESCRIPTION
**Pull request by kakirastern**
## Description

Experimented with `glue.core.coorindates`'s `WCSCoordinates` and leave comments or changed code snippets in the `glue-solar` codebase accordingly.

### CODE REVIEW

**Comment by kakirastern** [comment]
Will need a final check on the code soon... May need one more change.

**Comment by kakirastern** [comment]
PR ready for a review. (Turns out the potential change is not necessary.)

File: glue_solar/core/core.py [hunk 1]
```diff
@@ -27,7 +28,8 @@ def _parse_sunpy_map(data, label):
     scan_map = data
     label = label + '-' + scan_map.name
     result = Data(label=label)
-    result.coords = scan_map.wcs
+    result.coords = scan_map.wcs  # preferred way, preserves more info in some cases
+    # result.coords = WCSCoordinates(scan_map.fits_header)
```
Comment (nabobalis) [inline glue_solar/core/core.py]:
```suggestion
```

File: glue_solar/core/core.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 from glue.core import Component, Data
 from glue.core.visual import VisualAttributes
 from glue.core.data_factories import is_fits
+from glue.core.coordinates import WCSCoordinates
```
Comment (nabobalis) [inline glue_solar/core/core.py]: Is this import used anywhere?

File: glue_solar/core/sunpy_maps/loader.py [hunk 3]
```diff
@@ -77,7 +77,8 @@ def load_sunpy_map(self, sunpy_map):
             sunpy_map_loaded = sunpy.map.Map(sunpy_map)
             label = 'sunpy-map-' + sunpy_map_loaded.name
             data = Data(label=label)
-            data.coords = sunpy_map_loaded.wcs
+            data.coords = sunpy_map_loaded.wcs  # preferred way, preserves more info in some cases
+            # data.coords = WCSCoordinates(sunpy_map_loaded.fits_header)
```
Comment (nabobalis) [inline glue_solar/core/sunpy_maps/loader.py]:
```suggestion
```

File: glue_solar/instruments/iris/iris.py [hunk 4]
```diff
@@ -29,7 +29,7 @@ def _parse_iris_raster(data, label):
     for window, window_data in data.items():
         for i, scan_data in enumerate(window_data):
             w_data = Data(label=f"{window.replace(' ', '_')}-scan-{i}")
-            w_data.coords = scan_data.wcs
+            w_data.coords = WCSCoordinates(scan_data.header)
```
Comment (nabobalis) [inline glue_solar/instruments/iris/iris.py:32]: Why the change here?

Comment (kakirastern) [reply glue_solar/instruments/iris/iris.py:32]: I made the change here because I would like to use `WCSCoordinates` as far as possible, since it is native to `glue`. `WCSCoordinates` is built on top of `WCS`, so inherits everything from `WCS`, and more.

File: glue_solar/core/core.py (hunk 2 above)
Comment (kakirastern) [reply glue_solar/core/core.py]: Not anymore, and so I have removed the unused import statements.

**Comment by kakirastern** [comment]
PR is ready for another review.

**Review by nabobalis — APPROVED** [review]
